ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/MainActivity.kt

```diff
@@ -113,6 +117,10 @@ fun WearApp(watchRepository: WatchRepository) {
             currentBackStackEntry?.arguments?.getSerializable(SCROLL_TYPE_NAV_ARGUMENT)
                 ?: DestinationScrollType.NONE
 
+        // Display value is passed down to various user input screens, for the slider and stepper
+        // components specifically, to demonstrate how they work.
+        var displayValueForUserInput by remember { mutableStateOf(2) }
```

Comment:
Add TODO or at least extract `2` to defaultValue parameters, otherwise it's confusing.

Reply:
I added a TODO and also extracted 2 to defaultValue parameter

Reply:
I renamed it to defaultSelectedValue because it had a conflict with the defaultValue on line 244 which caused build failure when i ran gradle check